tools/releaseBuild/yaml/nuget.yml

```diff
@@ -13,15 +13,6 @@ steps:
     targetPath: "$(System.ArtifactsDirectory)/signed"
   displayName: Download signed windows native packages
 
-- powershell: |
-    $null = New-Item $(PackageRoot) -ItemType Directory -Force -Verbose
-    if(-not (Test-Path '$(System.ArtifactsDirectory)/release' ))
-    {
-      New-Item -ItemType Directory -Path '$(System.ArtifactsDirectory)/release' -Force
-    }
-    Invoke-WebRequest -Uri '$(PSRPBlobUrl)' -OutFile $(System.ArtifactsDirectory)/release/psrp.zip -Verbose
```

Comment:
@adityapatwardhan I'm not sure where the `PSRPBlobUrl` variable is defined. I can remove it if you point me to it.

Reply:
It is a variable in the variable group

Reply:
I know that, but don't know where that variable group exists.